ajax/libs/PKI.js/package.json

```diff
@@ -30,5 +30,6 @@
         ]
       }
     ]
-  }
+  },
+  "license": "MIT"
```

Comment:
It seems like it's `BSD-3-Clause`. 
ref: https://github.com/PeculiarVentures/PKI.js/blob/master/LICENSE, https://github.com/PeculiarVentures/PKI.js#license